doc/parallel_numpy.rst

```diff
@@ -23,6 +23,12 @@ worker processes.
   your code can release the GIL, then using ``backend="threading"`` is even
   more efficient.
 
+..
```

Comment:
Yes I thought that a user might refer to raw markup file too. Being a developer I have always seen raw markup files for documentation in my editor itself :sweat_smile: But eventually this documentation will be built and hosted in html format...